app/src/main/java/com/android/doctorapp/ui/appointment/AppointmentViewModel.kt

```diff
@@ -99,31 +100,35 @@ class AppointmentViewModel @Inject constructor(
 
     val symptomDetailsError: MutableLiveData<String?> = MutableLiveData()
     val sufferingDaysError: MutableLiveData<String?> = MutableLiveData()
+    val currentDate1: String = getCurrentDate()
 
     val isVisitedToggleData: MutableLiveData<Boolean> = MutableLiveData(false)
 
     var updateClick = MutableLiveData(false)
 
 
     private fun get15DaysList() {
-        val currentDate: String = getCurrentDate()
         val dateList = mutableListOf<Date>()
         val calendar = Calendar.getInstance()
-        calendar.time = dateFormatFull.parse(currentDate) as Date
+        calendar.time = dateFormatFull.parse(currentDate1) as Date
```

Comment:
Please give meaning full name, So that in future other developer can understand that used.

Reply:
Done